### DISCUSSION TITLE
I've tried to create custom transformer, but it was ignored :(

### DISCUSSION THREAD

**Discussion by gabplch** [rekalogika/mapper#94] [Q&A]
I tried to create a custom transformer, but it was ignored in `TransformerRegistry::findBySourceAndTargetType` during the mapping filter. I guess I skipped something or did something wrong with it.
This step also ignores build in `ObjectMapperTransformer`, which I've expected to use instead custom one :(
<img width="820" alt="image" src="https://github.com/user-attachments/assets/8c4488ad-c032-448d-b77f-a3dcc9708b79">

**Comment by priyadi** [comment]
Maybe try `bin/console cache:clear`, and see if it fixes your problem.

**Reply by gabplch** [reply]
Nope, I've tried to clear the cache and delete the var folder. I've also debagged all the way and created a conclusion that it's skipped because entry is not **Variant** and  not **SomewhatIdentical**

**Reply by priyadi** [reply]
in that case, can you create a minimal working reproducer for that?

**Reply by gabplch** [reply]
Ok, I'll try to debug more, maybe I missed something, and if problem won't be fixed, I'll let you know

**Reply by gabplch** [reply]
In general:
I have dto with the main object; let's name it `Car`. As to Car has an additional object field, named payload with some extra fields. This payload is abstract. One of them is SedanPayload, that have an InheritanceMap attribute from Dto to the Domain model.

I also have domain abstract `Car`, with concrete realizations (Sedan), which have fields, that are in SedanPayload.

Hope, it'll help you, but I'll still try debug it

**Reply by priyadi** [reply]
You should create a minimal reproducer example. You can add a test case to the mapper source code here, or alternatively, you can create a minimal Symfony app with a simple console command to run the case. What you said should be possible, but without a reproducer, it is difficult to pinpoint where the problem is at.
